components/region_cache_memory_engine/src/lib.rs

```diff
@@ -99,3 +104,21 @@ impl RangeCacheEngineConfig {
         }
     }
 }
+
+pub struct RangeCacheEngineOptions {
```

Comment:
The naming of RangeCacheEngineOptions is not appropriate. Statistics should not be part of Option.

Reply:
I think statistics should be just a member of RangeCacheMemoryEngine directly and initialized inside its constructor.